packages/plugin-react/src/index.ts

```diff
@@ -281,7 +303,10 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
           sourceMaps: true,
           // Vite handles sourcemap flattening
           inputSourceMap: false as any
-        })
+        }
+        delete (transformOptions as any).ssr
+        delete (transformOptions as any).file
```

Comment:
Maybe we could avoid the deletes using destructuring? Something like:
```js
        const { ssr, file, ...babelOptions } = reactBabelOptions   
        const transformOptions: TransformOptions = {
          ...babelOptions,
          ast: !isReasonReact,
          root: projectRoot,
	        sourceMaps: true,
          // Vite handles sourcemap flattening
          inputSourceMap: false as any
        }
```

Reply:
Done